Album/ViewControllers/MapControllers/FriendMapViewController.m

```diff
@@ -0,0 +1,237 @@
+//
+//  FriendMapViewController.m
+//  Album
+//
+//  Created by Airei Fukuzawa on 7/18/22.
+//
+
+#import "FriendMapViewController.h"
+#import "LocationGenerator.h"
+#import "ColorConvertHelper.h"
+#import "DetailsViewCOntroller.h"
+#import "InfoPOIView.h"
+#import "InfoMarkerView.h"
+#import "AlbumConstants.h"
+#import "Parse/Parse.h"
+#import <Parse/PFImageView.h>
+#import "Pin.h"
+
+@interface FriendMapViewController () <GMSMapViewDelegate, GMSIndoorDisplayDelegate, CLLocationManagerDelegate>
+@property (nonatomic, strong) NSMutableArray *markerArr;
+@property (nonatomic, strong) NSMutableDictionary *placeToPins;
+@property (nonatomic, strong) NSMutableDictionary *pinImages;
+@property (nonatomic, strong) NSDateFormatter *formatter;
+@property (nonatomic, strong) ColorConvertHelper *colorHelper;
+@end
+
+@implementation FriendMapViewController
+
+- (void)loadView {
```

Comment:
Lets try to use `viewDidLoad`

<img width="716" alt="Screen Shot 2022-07-22 at 10 19 11 AM" src="https://user-images.githubusercontent.com/21207111/180491505-3244ea3f-0316-4c77-97fa-cd8cc7f3a724.png">

https://medium.com/good-morning-swift/ios-view-controller-life-cycle-2a0f02e74ff5